.github/dependabot.yml

```diff
@@ -0,0 +1,20 @@
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+
+  # Rust projects
+  - package-ecosystem: "cargo"
+    directories:
+      - "/"
+      - "/wayshot"
+      - "/libwayshot"
+      - "/libwayshot/examples/waymirror-egl"
+    schedule:
+      interval: "daily"
```

Comment:
Daily is too frequent.. I think one time one week will be ok

Reply:
Yeah, makes sense.